components/Starknet/modules/tools/pages/starkgate_dev_reference.adoc

```diff
@@ -0,0 +1,490 @@
+[id="StarkGate_function_reference"]
+= StarkGate API reference
+
+The StarkGate smart contracts include functions that you use to implement various flows in a dApp.
+
+The L1 functions are defined in the following smart contracts:
+
+[horizontal]
+link:https://github.com/starkware-libs/starkgate/blob/dev/src/solidity/IStarkgateManager.sol[IStarkgateManager.sol]:: The StarkGate Manager API
+link:https://github.com/starkware-libs/starkgate/blob/dev/src/solidity/IStarkgateRegistry.sol[IStarkgateRegistry.sol]:: The StarkGate Registry API
+link:https://github.com/starkware-libs/starkgate/blob/dev/src/solidity/StarknetTokenBridge.sol[StarkGateTokenBridge.sol]:: The StarkGate bridge API
```

Comment:
A short description of what's the contract used for would be more useful than just re writing the contract's name

Reply:
Do you have any other look and feel suggestions (in general)? It would be good to create an issue for this with a label like **ui** or **look and feel**.

Reply:
not sure i'll have the time to do it today but will make sure to do it for next ones